Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionPublishToQCOperations.cs

```diff
@@ -82,5 +119,57 @@ public void Execute(IReportInfo RI)
             }
         }
 
+        private BusinessFlow ConvertRunSetToBF(IReportInfo reportInfo)
+        {
+            RunsetExecutor runSetExec = WorkSpace.Instance.RunsetExecutor;
+            try
+            {
+                if (reportInfo == null)
+                {
+                    return null;
+                }
+
+                //Create Business Flow
+                BusinessFlow virtualBF = new BusinessFlow();
+                virtualBF.Name = runSetExec.RunSetConfig.Name;
+                virtualBF.Description = runSetExec.RunSetConfig.Description;
+                virtualBF.Status = BusinessFlow.eBusinessFlowStatus.Development;
```

Comment:
what this 'Status' field used for in EXport?
why to set the default value to 'Development' and not to 'Automated'?

Reply:
not in use, we don't have automated value, most of the BF status is Unknown.